### PR TITLE
Updated ALCARECO event content: more esplicit list of collections, ma…

### PR DESCRIPTION
**Pull request by shervin86**
marginal change in list of collections
As requested by David some time ago, I made the list of keep with less *.

Fixing also one missing SuperCluster collection

@emanueledimarco @ldcorpe @lbrianza want to follow this PR

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild please test

File: Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py [hunk 1]
```diff
@@ -2,18 +2,6 @@
 
 # output block for alcastream Electron
 OutALCARECOEcalCalElectron_specific = cms.untracked.vstring(
```
Comment (mmusich) [inline Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py]: what is the purpose of this now that is empty?

Comment (shervin86) [inline Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py]: ... forgot to remove...

--
                          Shervin
Da: Marco Musich
Inviato: lunedì 28 settembre 2015 14:53
A: cms-sw/cmssw
Rispondi a: cms-sw/cmssw
Cc: Shervin Nourbakhsh
Oggetto: Re: [cmssw] Updated ALCARECO event content: more esplicit list of collections, ma… (#11522)



In Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py<https://github.com/cms-sw/cmssw/pull/11522#discussion_r40547692>:

> @@ -2,18 +2,6 @@
>
>  # output block for alcastream Electron
>  OutALCARECOEcalCalElectron_specific = cms.untracked.vstring(


what is the purpose of this now that is empty?

**Comment by shervin86** [comment]
Updated branch removing the unuseful line

**Comment by mmusich** [comment]
please test

**Comment by shervin86** [comment]
any news about that?

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
@shervin86 - seems in places there are more * - is there some reason not to be able to specify what module is providing the data product?

**Comment by shervin86** [comment]
Dear Dave,
I have fixed the part related to ecal clusters and superClusters at my best, but this is not obvious in some cases where collections are linked to others. For example in the electron collection some electrons are linked to a superCluster collection and other electrons to another superCluster collection. Some of the * are there to keep as many collections needed without having to run on a very large dataset to identify all of them. Does this represent a problem? Do you need I reduce more the number of *? I will not remove all of them anyway.

**Comment by davidlange6** [comment]
@shervin86

you are adding *s in this PR, thus my concern… the best approach is to take a file from your alcarecos from prompt do edmEventSize -v <file> and then do something like

cat output | awk ‘{print $1}’ | tr ‘_’ ‘ ‘ | awk ‘{print “keep "$1”_”$2”_*_*,”}’ 

or something like that to get a list of all keep statements you need. I don’t know what you mean about collections linked to others.

**Comment by davidlange6** [comment]
I am cleaning up the 76x queue aside from things for analysis workflows. I'm closing this pull request, please make sure the PR is in 80x. Thanks!

**Pull request closed without merging**